Headers/DebugServer2/Target/Linux/Process.h

```diff
@@ -63,11 +63,11 @@ class Process : public ds2::Target::POSIX::ELFProcess {
   ErrorCode writeCPUState(ThreadId tid, Architecture::CPUState const &state,
                           uint32_t flags = 0);
 
-#if defined(ARCH_ARM) || defined(ARCH_ARM64)
+#if defined(ARCH_ARM)
```

Comment:
Because the arm64 version used to always return 0 (which is taken care of
by the implementation in the base class).

On Thursday, April 28, 2016, strager <notifications@github.com> wrote:

> In Headers/DebugServer2/Target/Linux/Process.h
> <https://github.com/facebook/ds2/pull/304#discussion_r61447268>:
>
> > @@ -63,11 +63,11 @@ class Process : public ds2::Target::POSIX::ELFProcess {
> >    ErrorCode writeCPUState(ThreadId tid, Architecture::CPUState const &state,
> >                            uint32_t flags = 0);
> >
> > -#if defined(ARCH_ARM) || defined(ARCH_ARM64)
> > +#if defined(ARCH_ARM)
>
> Why drop ARM64?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/facebook/ds2/pull/304/files/50969482a1682555a58812a9b02e3b764822fab7..f40e4e139f23c07a172d57f185e36ce483b55024#r61447268>
>


-- 
Stephane Sezer